src/main/groovy/com/avioconsulting/mule/linter/model/ProjectFile.groovy

```diff
@@ -0,0 +1,37 @@
+package com.avioconsulting.mule.linter.model
+
+class ProjectFile {
+
+    String name
+    String type
```

Comment:
These are never set by the PomFile, what are they for?

Reply:
Not used yet, added for future use. Removing now we will add in future when needed.